composer.json

```
@@ -70,8 +71,10 @@
         ],
         "ci:php:fixer": "\"./.phive/php-cs-fixer\" --config=config/php-cs-fixer.php fix --dry-run -v --show-progress=dots --diff bin src tests",
         "ci:php:stan": "phpstan --no-progress --configuration=config/phpstan.neon",
+        "ci:php:rector": "\"./vendor/bin/rector\" --no-progress-bar --dry-run --config=config/rector.php",
```

For binaries supplied by Composer packages, we can use them directly in Composer commands without the `vendor/bin` path:

```suggestion
        "ci:php:rector": "rector --no-progress-bar --dry-run --config=config/rector.php",
```